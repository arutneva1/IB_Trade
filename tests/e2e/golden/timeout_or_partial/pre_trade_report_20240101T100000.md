NetLiq: 10000.00
Cash USD: 10000.00
Cash Buffer: 100.00

| symbol | target_pct | current_pct | drift_bps | price | dollar_delta | share_delta | side | est_notional | reason |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| AAA | 100.00 | 0.00 | 10000.00 | 100.00 | 10000.00 | 100.0000 | BUY | 10000.00 |  |
| TOTAL | 100.00 | 0.00 | 10000.00 |  | 10000.00 |  |  | 10000.00 |  |
