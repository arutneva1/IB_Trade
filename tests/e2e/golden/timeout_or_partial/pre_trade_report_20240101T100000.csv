NetLiq,10000.00
Cash USD,10000.00
Cash Buffer,100.00

symbol,target_pct,current_pct,drift_bps,price,dollar_delta,share_delta,side,est_notional,reason
AAA,100.0,0.0,10000.0,100.0,10000.0,100.0,BUY,10000.0,
TOTAL,100.0,0.0,10000.0,,10000.0,,,10000.0,
